README.md

```diff
@@ -55,6 +53,14 @@ console.log(errors.AccessDenied);
 
 ```
 
+## Build Status
+
+![badgepub]
+
+## Codecov Status
+
+![codecov]
```

Comment:
i think it's the usual practice to keep the badges "displayed" on top of the README file, so they are visible when we open the github repository's page